### PR TITLE
potential bug fix in dicom pln import

### PR DESCRIPTION
**Pull request by markbangert**
@AlexS1992 could you please have a look real quick? Today we had issues reading in a dicom plan object that was exported with the raystation. I think the problem is that not for every control point there must be information about the jaws.

### CODE REVIEW

File: dicomImport/matRad_importFieldShapes.m [hunk 1]
```diff
@@ -80,54 +80,66 @@
...
            for k = 1:length(currDeviceSeqNames)
+           
                % beam limiting device position sequence has to be defined on
                % the first control point and has to be defined on following
-               % points only if it changes
-               currLeafPos = currControlPointElement.BeamLimitingDevicePositionSequence.(currDeviceSeqNames{k}).LeafJawPositions;          
+               % points only if it changes -> default initilation if counter > 1
+               if counter > 1
+                   tmpCollimation.Fields(counter).LeafPos{k}(:,1) = tmpCollimation.Fields(counter-1).LeafPos{k}(:,1);
+                   tmpCollimation.Fields(counter).LeafPos{k}(:,2) = tmpCollimation.Fields(counter-1).LeafPos{k}(:,2);
```
Comment (AlexS1992) [inline dicomImport/matRad_importFieldShapes.m]: Maybe combine these 2 lines into
tmpCollimation.Fields(counter).LeafPos{k} = tmpCollimation.Fields(counter-1).LeafPos{k}; ?

File: dicomImport/matRad_importFieldShapes.m [hunk 2]
```diff
@@ -80,54 +80,66 @@
...
+                   currLeafPos = currControlPointElement.BeamLimitingDevicePositionSequence.(currDeviceSeqNames{k}).LeafJawPositions;          
+
+                   deviceIx = find(strcmp({device(:).DeviceType}, ...
+                       currControlPointElement.BeamLimitingDevicePositionSequence.(currDeviceSeqNames{k}).RTBeamLimitingDeviceType));
 
-               if (length(currLeafPos) ~= 2 * device(k).NumOfLeafs)
-                   warning(['Number of leafs/jaws does not match given number of leaf/jaw positions in control point sequence ' ...
-                            currControlPointSeqNames{j} ' on beam sequence ' BeamSeqNames{i} ' for device ' ...
-                            device(k).DeviceType '. No field shape import performed!']);
-                   return;
```
Comment (AlexS1992) [inline dicomImport/matRad_importFieldShapes.m]: Is there a reason why you changed the return + warning statement into an error?
If you prefer errors I think we should use these consistent in all places.

**Comment by AlexS1992** [comment]
@markbangert Just to understand the issue better:
The problem occured for a control point where 'BeamLimitingDevicePositionSequence' was defined, but not all of the devices, which where defined in the first control point, had an element in the BeamLimitingDevicePositionSequence. Is that right?

**Comment by markbangert** [comment]
I think I like errors better. When something does not work as anticipated I
think the code should blow up. Only th GUI should remain stable. But I dont
think there is an urgent need to fix this globally right away...

**Comment by markbangert** [comment]
Yes. You understood correctly.

**Comment by AlexS1992** [comment]
From reading the DICOM documentation a bit I thought the whole BeamLimitingDevicePositionSequence had to be repeated if one element changed, but apparently this is not the case...

Your changes are fine by me, I can see no problems in adding this.

File: dicomImport/matRad_importFieldShapes.m (hunk 1 above)
Comment (markbangert) [inline dicomImport/matRad_importFieldShapes.m]: Yes.

File: dicomImport/matRad_importFieldShapes.m (hunk 2 above)
Comment (markbangert) [inline dicomImport/matRad_importFieldShapes.m]: I just changed my mind on this. I figure it is a good thing that the import at least works for the other structures..

**Pull request merged**